tests/fixtures/expected_prometheus_metrics.prom

```diff
@@ -19,6 +19,10 @@ tx_queue 2.0
 # HELP block_template_error Number of errors updating block template
 # TYPE block_template_error gauge
 block_template_error 1.0
+# HELP miner_rtt Miner RTT
+# TYPE miner_rtt gauge
+miner_rtt{miner_address="abc123"} 0.0
```

Comment:
Address as in wallet address? Or uid? I guess there might be multiple connections using the same wallet address which might make things confusing.

Reply:
I guess we can add the `miner_id` as another label to all metrics that currently use only `miner_address`.

Adding labels won't disrupt our charts in Grafana.

Reply:
Done to all metrics that use `miner_address`

https://github.com/HathorNetwork/tx-mining-service/pull/114/commits/094b75aa15affb9d878eef615e32162f3749cf52

Reply:
Thanks!